templates/openstack/index.html

```diff
@@ -0,0 +1,457 @@
+{% endblock meta_copydoc %}
+
+{% block title %}Canonical OpenStack{% endblock %}
+
+{% block meta_description %}
+  Canonical OpenStack - your own professional cloud for sophisticated workloads, including AI/ML, and full control over your sensitive data
+{% endblock %}
+
+{% block body_class %}
+  is-paper
+{% endblock body_class %}
+
+{% block content %}
+
+  <section class="p-section--hero">
+    <div class="row--50-50">
+      <div class="col">
+        <div class="p-section--shallow">
+          <h1>Canonical OpenStack</h1>
+        </div>
+      </div>
+      <div class="col">
+        <p>
+          Get a public cloud-like experience while hosting your applications in your own data centre. Strengthen your business with a flexible infrastructure that meets the highest industry standards.
+        </p>
+        <div class="p-cta-block">
+          <a href="https://microstack.run/docs/single-node"
+             class="p-button--positive">Try Canonical OepnStack</a>
+          <a href="https://ubuntu.com/engage/openstack-ebook-beginners"
+             class="p-button">Download OpenStack e-book</a>
+        </div>
+      </div>
+    </div>
+  </section>
+
+  <section class="p-section">
+    <div class="row">
+      <hr class="u-no-margin--bottom" />
+      <div class="col-3 p-section--shallow">
+        <h2 class="p-text--small-caps">Canonical OpenStack in numbers</h2>
+      </div>
+      <div class="col-9">
+        <div class="row">
+          <div class="col-3 col-medium-2">
+            <hr class="p-rule--highlight" />
+            <p class="p-heading--2">
+              <strong>100&plus;</strong>
+            </p>
+            <p class="p-heading--4 u-no-padding--top">Happy customers</p>
+          </div>
+          <div class="col-3 col-medium-2">
+            <hr class="p-rule--highlight" />
+            <p class="p-heading--2">
+              <strong>500&plus;</strong>
+            </p>
+            <p class="p-heading--4 u-no-padding--top">Clouds in production</p>
+          </div>
+          <div class="col-3 col-medium-2">
+            <hr class="p-rule--highlight" />
+            <p class="p-heading--2">
+              <strong><span aria-label="about">~</span>30 minutes</strong>
+            </p>
+            <p class="p-heading--4 u-no-padding--top">To spin it up in your lab</p>
+          </div>
+        </div>
+      </div>
+    </div>
+  </section>
+
+  <section class="p-section">
+    <div class="row--50-50">
+      <hr class="p-rule" />
+      <div class="col">
+        <h2>Keep your future in your hands with your own professional cloud</h2>
+      </div>
+      <div class="col">
+        <p>
+          Transform your data centre into a fully functional and professional cloud environment. Take advantage of cloud-based operations, while retaining all the benefits of owning your computing infrastructure.
+        </p>
+        <div class="p-cta-block">
+          <a class="p-button--positive">Get in touch</a>
```

Comment:
This and any other CTA that is supposed to link to this form will not work for now. See discussion [here](https://warthogs.atlassian.net/browse/WD-14071?focusedCommentId=597407)